src/server.js

```diff
@@ -1189,6 +1222,36 @@ async function startPolkadotAPI() {
                 {'error': e.toString()});
         }
     });
+
+    // Session
```

Comment:
Again kindly put this before the Derive Staking one to preserve the alphabetical order